src/Microsoft.DotNet.Interactive.FSharp.Tests/KernelTests.fs

```diff
@@ -44,6 +44,24 @@ type KernelTests() =
             
             texts
 
+    let getCompletions (line : int) (column : int) (code : #seq<string>)  =
+        let code = String.concat "\r\n" code
```

Comment:
We probably won't want the `\r` on non-Windows OSs, yeah?

Reply:
Probably not, good catch